.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required 
+dist: trusty 
+language: generic 
+compiler:
+  - gcc
+notifications:
+  email:
+    on_success: always
+    on_failure: always
+#    recipients:
```

Comment:
I like this feature as well, but I'm already overloaded with email, so I'm going to leave it "off" for now.